piccolo/columns/column_types.py

```diff
@@ -77,6 +80,23 @@ class Band(Table):
     from piccolo.table import Table
 
 
+class ColumnKwargs(t.TypedDict, total=False):
+    null: bool
+    primary_key: bool
+    primary: bool
+    key: bool
```

Comment:
I would drop these two. We don't want people passing them in any more.

Reply:
I would remove it if you agree to change this test.

https://github.com/piccolo-orm/piccolo/blob/1e69d7b16bad0a31ba329e79581723f95df5195b/tests/columns/test_primary_key.py#L77-L85

to `primary_key=True`. Otherwise we get a Mypy error.

Reply:
I made a new commit with the latest changes. I hope it's okay.